src/modules/common-elements/constants.ts

```diff
@@ -443,6 +466,7 @@ export const CANCELED = "CANCELED";
 export const OPEN = "OPEN";
 export const PRICE = "price";
 export const SHARE = "share";
+export const SHARES = "Shares";
```

Comment:
Any reason why this is sentence case but the others above are lowercase?

Reply:
Thought I’d cap it here. Since the other value used in the dropdown is all case. So they arent consistent.